src/main/java/com/ericsson/ei/frontend/WebController.java

```diff
@@ -16,75 +16,59 @@
 */
 package com.ericsson.ei.frontend;
 
-import com.ericsson.ei.frontend.model.BackEndInformation;
-import com.ericsson.ei.frontend.utils.BackEndInstancesUtils;
+import com.ericsson.ei.frontend.utils.WebControllerUtils;
 
 import javax.servlet.http.HttpServletRequest;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.RequestMapping;
 
-import lombok.Setter;
-
 @Controller
 public class WebController {
 
-    @Value("${ei.frontendServiceHost}")
-    private String frontendServiceHost;
-
-    @Setter
-    @Value("${ei.frontendServicePort}")
-    private int frontendServicePort;
-
-    @Value("${ei.frontendContextPath}")
-    private String frontendContextPath;
-
-    @Value("${ei.eiffelDocumentationUrls}")
-    private String eiffelDocumentationUrls;
-
-    @Value("${ei.useSecureHttpFrontend}")
-    private boolean useSecureHttpFrontend;
-
     @Autowired
-    private BackEndInstancesUtils backEndInstancesUtils;
+    private WebControllerUtils frontEndUtils;
 
+    public static final Logger LOG = LoggerFactory.getLogger(WebController.class);
     @RequestMapping("/")
     public String greeting(Model model) {
-
-        model.addAttribute("frontendServiceUrl", getFrontendServiceUrl());
-        String eiffelDocumentationUrlLinks = String.format("%s", eiffelDocumentationUrls);
+    	LOG.error("FRONTEND: " + frontEndUtils.getFrontEndServiceUrl());
+        model.addAttribute("frontendServiceUrl", frontEndUtils.getFrontEndServiceUrl());
+        String eiffelDocumentationUrlLinks = String.format("%s", frontEndUtils.getEiffelDocumentationUrls());
         model.addAttribute("eiffelDocumentationUrlLinks", eiffelDocumentationUrlLinks);
         return "index";
     }
 
     @RequestMapping("/subscriptionpage.html")
     public String subscription(Model model) {
-
-        model.addAttribute("frontendServiceUrl", getFrontendServiceUrl());
+    	LOG.error("FRONTEND: " + frontEndUtils.getFrontEndServiceUrl());
```

Comment:
Why error here?,,, nothing fails here from what I can see.
Same comment for the rest of the log errors in this class.

Reply:
The logging was just used for debug while testing without activating debug logging. Will be removed.